clouddriver-appengine/src/main/groovy/com/netflix/spinnaker/clouddriver/appengine/deploy/validators/StartAppEngineDescriptionValidator.groovy

```diff
@@ -0,0 +1,27 @@
+/*
+ * Copyright 2016 Google, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.netflix.spinnaker.clouddriver.appengine.deploy.validators
+
+import com.netflix.spinnaker.clouddriver.appengine.AppEngineOperation
+import com.netflix.spinnaker.clouddriver.orchestration.AtomicOperations
+import org.springframework.stereotype.Component
+
+@AppEngineOperation(AtomicOperations.START_SERVER_GROUP)
+@Component("startAppEngineDescriptionValidator")
```

Comment:
You should verify this, but I think you can just use `@Component` without the string parameter (since you've used `@AppEngineOperation(AtomicOperations.START_SERVER_GROUP)`).